optimum/intel/openvino/quantization.py

```diff
@@ -169,6 +173,7 @@ def _onnx_export(model: NNCFNetwork, config: OnnxConfig, model_inputs: Dict, f:
             )
         max_onnx_opset = min(config.default_onnx_opset, MAX_ONNX_OPSET)
         opset = max_onnx_opset if _openvino_version > version.Version("2022.2.0") else MAX_ONNX_OPSET_2022_2_0
+        opset = opset if not ov_config.dump_onnx else max(opset, MIN_ONNX_QDQ_OPSET)
```

Comment:
If the opset is set to `MIN_ONNX_QDQ_OPSET` instead of `MAX_ONNX_OPSET_2022_2_0` wouldn't that cause any issue for the resutling OpenVINO model ?

Reply:
It could but the user should not use save_onnx_model option in this case. This is because per-channel QuantizeLinear/DequantizeLinear is supported starting from  opset13.

Reply:
The problem should be solved in the next openvino release where the min ONNX opset will be 13 right ? In the meantime we can either add a warning to explain it or even remove the export to the openvino format in the case where `save_onnx_model` is set to `True`.

Reply:
I am ok with the warning but a new OV version is coming soon (pre-release is available) so let's keep export to OV as well.

Reply:
We can add a logger warning explaning the potential risks encountered if `MIN_ONNX_QDQ_OPSET` is strictly higher than `MAX_ONNX_OPSET` / `MAX_ONNX_OPSET_2022_2_0` (depending on the `openvino` version).

Reply:
To tell the truth, the current statement that `MAX_ONNX_OPSET_2022_2_0=10` is not fully correct and it is caused by the fact that some models may not work while OpenVINO 2022.2 supporst ONNX opset higher than 13. This is the local issue that will be fixed in 22.3 soon.
I updated the warnings. Please check.